CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ endif()
 
 #--- Declare options -----------------------------------------------------------
 option(CREATE_DOC "Whether or not to create doxygen doc target." OFF)
+option(SIO_HANDLERS "Build the SIO block handlers and readers and writers." OFF)
```

Comment:
naming a bit too technical. Why not "SIO_SUPPORT" or "ENABLE_SIO" ? One can use equivalent naming for every further backend.

Reply:
I have changed it to `ENABLE_SIO`. I realized that this has to be done all over the place, so we should come up with a more "compact" way of adding backends.

Reply:
yap. that was my comment in the meeting already. thanks!

Reply:
I think we can broaden the discussion in #131 to cover the whole topic of how to best deal with different I/O backends.